docs/customization/theme.md

```diff
@@ -1,21 +1,15 @@
-# Theme Creation, Overrides, and Assets
+# Theme Creation, Overrides, and Static Files
 
 Unless if you are happy with the out-of-the-box Open ONI look, you will probably want a theme!  Open ONI ships with a default theme (`/themes/default`) that you may want to refer to while you work on your own.
 
-Quick reference to update static files (CSS, images, etc) after making changes in the docker dev environment:
-
-```
-docker-compose exec web manage collectstatic --noinput
-```
-
 Collected static files are stored in the `/static/compiled/` directory. You
-will need to compile static assets after making changes to CSS and JS.
+will need to compile static files after making changes to CSS and JS.
```

Comment:
These two lines should probably be integrated into the text below the heading. Seems out of place left here.